internal/pkg/cli/interfaces.go

```diff
@@ -166,6 +166,19 @@ type environmentDeployer interface {
 	GetEnvironment(projectName, envName string) (*archer.Environment, error)
 }
 
+type appDeployer interface {
+	// DeployApp // TODO ADD
+	DeleteApp(in deploy.DeleteAppInput) error
+}
+
+type appRemover interface {
+	RemoveAppFromProject(project *archer.Project, appName string) error
+}
```

Comment:
Should we move `DeleteApp` here?
```suggestion
type appRemover interface {
	DeleteApp(in deploy.DeleteAppInput) error
	RemoveAppFromProject(project *archer.Project, appName string) error
}
```

Reply:
Hm, I thought about this but hesitated since the appDeployer is used as the return value for the getAppDeployer function, which uses a different session each time, whereas the appRemover uses the default session so I was nervous about the coupling. What do you think?